src/devices/bus/a2bus/uthernet.cpp

```diff
@@ -0,0 +1,77 @@
+// license:BSD-3-Clause
+// copyright-holders:R. Belmont
+/*********************************************************************
+
+    uthernet.cpp
+
+    Apple II Uthernet Card
+
+*********************************************************************/
+
+#include "emu.h"
+#include "emuopts.h"
+#include "uthernet.h"
+
+/***************************************************************************
+    PARAMETERS
+***************************************************************************/
+
+//**************************************************************************
+//  GLOBAL VARIABLES
+//**************************************************************************
+
+DEFINE_DEVICE_TYPE(A2BUS_UTHERNET, a2bus_uthernet_device, "uthernet", "Uthernet")
+
+//**************************************************************************
+//  LIVE DEVICE
+//**************************************************************************
+
+a2bus_uthernet_device::a2bus_uthernet_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock) :
+		a2bus_uthernet_device(mconfig, A2BUS_UTHERNET, tag, owner, clock)
+{
+}
+
+a2bus_uthernet_device::a2bus_uthernet_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock) :
+		device_t(mconfig, type, tag, owner, clock),
+		device_a2bus_card_interface(mconfig, *this),
+		m_started(false),
+		m_interface(mconfig.options().value(OPTION_UTHERNET_INTF)),
+		m_netinf(*this, "cs8900a")
+{
+}
+
+void a2bus_uthernet_device::device_add_mconfig(machine_config &config)
+{
+	CS8900A(config, m_netinf, 0);
```

Comment:
Is the Ethernet MAC not clocked?  If you know what its clock input is, it’s best to include it for documentation purposes, even if it isn’t used yet.